.github/workflows/code-and-dependencies-checks.yml

```diff
@@ -13,18 +13,30 @@ jobs:
         uses: chartboost/ruff-action@v1
         with:
           args: check
-  poetry-checks:
+  poetry-dependencies-and-Django-code-tests:
```

Comment:
Couldn't you have come up with even a longer name?

Reply:
then how should I name it?

Reply:
code-tests/code-checks

Reply:
but id doesn't only check code, but also dependencies

Reply:
That name is readable and pretty good. Don't listen to this Java stan, Kubuś. You could maybe think of something shorter, but I have no ideas.

Reply:
Why do you want to include `Django` and `poetry` in the name?

Reply:
I thought it was better if name said what is used to do job, like with first job name where ruff is included